presto-hive/src/test/java/com/facebook/presto/hive/TestHiveUtil.java

```diff
@@ -156,9 +156,6 @@ private static void assertToPartitionNamesAndValues(String partitionName, List<S
 
         for (int index = 0; index < expectedKeyList.size(); index++) {
             String key = expectedKeyList.get(index);
-            if (!actual.containsKey(key)) {
```

Comment:
so if the expected key is not present in the partition name, we simply ignore it? Also this makes followup check redundant.

Lets keep this change in the separate PR?

Reply:
@jainxrohit If an expected key is not present in the partition name, then the checkState will throw the error. I'll keep this in a separate PR.